src/EditorFeatures/Test/CodeFixes/ExtensionOrderingTests.cs

```diff
@@ -17,6 +17,7 @@ public class ExtensionOrderingTests
         private readonly ExportProvider _exportProvider = TestExportProvider.ExportProviderWithCSharpAndVisualBasic;
 
         [Fact]
+        [Trait("Require32", "true")]
```

Comment:
Didn't bother to investigate deeply.  If I couldn't find it out in a few minutes I just disabled it.